name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/comdirect/ComdirectPDFExtractorTest.java

```diff
@@ -1483,29 +1483,29 @@ public void testDividendeAusSteuermitteilung3()
     public void testDividendeAusSteuermitteilung4()
     {
         ComdirectPDFExtractor extractor = new ComdirectPDFExtractor(new Client());
-
+    
         List<Exception> errors = new ArrayList<>();
-
+    
```

Comment:
Whitespace in empty lines (happens 8 times in this file).

Reply:
Fun fact: this "feature" can be reproduced. I didn't really touch that piece of code, only moved it in the "Outline" view, what seems to create those whitespaces in all empty lines of the block moved.